lib/appsignal/plug.ex

```diff
@@ -48,26 +48,25 @@ if Appsignal.plug?() do
       handle_error(conn, kind, reason, reason, stack)
     end
 
+    def handle_error(_conn, kind, %{plug_status: status} = reason, _wrapped_reason, stack)
+        when status < 500 do
```

Comment:
what's the behavior here? We do catch errors below code 500 now, right? so why do we need to reraise it?

Reply:
We don't in this patch. After all three are merged, the behaviour should be the same as before, without < 500 errors being caught by the ErrorHandler and ending up in AppSignal (#417). We'll make a decision on more flexible filtering in #410.

Reply:
Ok, so we don't want to fix that issue (not tracking non-500 status code) errors in these PRs just yet? Just want to make sure

Reply:
That's correct. We'll tackle that in #410.